.github/workflows/int-and-connected-test-run.yml

```
@@ -244,6 +244,10 @@ jobs:
         os: [ubuntu-latest]
     if: always()
     runs-on: ${{ matrix.os }}
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+      RUN_STATUS: ${{ (needs.test_check.outputs.job-status == 'success' && needs.test_connected.outputs.job-status == 'success' && needs.deploy_test_integration.outputs.job-status == 'success') && 'success' || 'failure' }}
```

This is a good idea to use an env var to store the aggregated test result. If you want, you can use a yaml block scalar to format this access lines. https://yaml-multiline.info
```suggestion
      RUN_STATUS: >-
        ${{ (needs.test_check.outputs.job-status == 'success'
        && needs.test_connected.outputs.job-status == 'success' 
        && needs.deploy_test_integration.outputs.job-status == 'success') 
        && 'success' || 'failure' }}
 ```

---

Nice, I like this!